src/cares_wrap.cc

```diff
@@ -50,6 +50,7 @@
 #endif
 
 #if defined(__OpenBSD__)
+# define AI_ALL 0
 # define AI_V4MAPPED 0
```

Comment:
Obviously, this was this way before, but I'm wondering why we're not doing

```suggestion
// OpenBSD does not define these
#ifndef AI_ALL
#define AI_ALL 0
#endif
#ifndef AI_V4MAPPED
#define AI_V4MAPPED 0
```

which seems just a bit more future-proof (because OpenBSD might add support at some point) and flexible (because what if there's other OSes that also don't have these)?

Reply:
Looks good to me. From what I can tell, that change should apply to 14.x as well, no?

Reply:
The current version in OpenBSD ports is 12.16.1 and I was trying to upgrade that (I'm just a user and not involved in the project), so my concern was mainly to get a more recent 12.x version to build on account of the numerous security patches. 

I can confirm that with the current patches in the port and this fix, I get a clean build and as far as I can tell a working node environment on OpenBSD, but I have not even looked into building any later release branch, let alone MASTER yet, so I thought it'd be best to stick to the branch I've actually tested :-)